drizzle-orm/src/pg-core/columns/postgis_extension/geometry.ts

```diff
@@ -46,7 +58,12 @@ export class PgGeometry<T extends ColumnBaseConfig<'array', 'PgGeometry'>> exten
 	}
 
 	override mapToDriverValue(value: [number, number]): string {
-		return `point(${value[0]} ${value[1]})`;
+		const point = `point(${value[0]} ${value[1]})`;
+
+		if (this.srid) {
+			return `ST_GeomFromText('${point}', ${this.srid})`;
```

Comment:
We probably want to make this `ST_SetSRID(ST_MakePoint(${value[0], ${value[1]}), ${this.srid})` or code to that effect

Reply:
Can confirm `ST_SetSRID(ST_MakePoint(${value[0], ${value[1]}), ${this.srid})` is the correct way to handle this.